fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -9276,23 +9326,23 @@
 ## `selected` (on `<option>` inside `<select>`)
 | Test Case | Flags | Result |
 | --- | --- | --- |
-| `selected=(string)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(empty string)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(array with string)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(empty array)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(object)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(numeric string)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(-1)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(0)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(integer)`| (initial, warning, ssr warning)| `<boolean: true>` |
+| `selected=(string)`| (initial, warning)| `<boolean: true>` |
+| `selected=(empty string)`| (initial, warning)| `<boolean: true>` |
+| `selected=(array with string)`| (initial, warning)| `<boolean: true>` |
+| `selected=(empty array)`| (initial, warning)| `<boolean: true>` |
+| `selected=(object)`| (initial, warning)| `<boolean: true>` |
+| `selected=(numeric string)`| (initial, warning)| `<boolean: true>` |
+| `selected=(-1)`| (initial, warning)| `<boolean: true>` |
+| `selected=(0)`| (initial, warning)| `<boolean: true>` |
+| `selected=(integer)`| (initial, warning)| `<boolean: true>` |
 | `selected=(NaN)`| (initial, warning)| `<boolean: true>` |
-| `selected=(float)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(true)`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(false)`| (initial, warning, ssr warning)| `<boolean: true>` |
+| `selected=(float)`| (initial, warning)| `<boolean: true>` |
+| `selected=(true)`| (initial, warning)| `<boolean: true>` |
+| `selected=(false)`| (initial, warning)| `<boolean: true>` |
 | `selected=(string 'true')`| (initial, warning)| `<boolean: true>` |
 | `selected=(string 'false')`| (initial, warning)| `<boolean: true>` |
-| `selected=(string 'on')`| (initial, warning, ssr warning)| `<boolean: true>` |
-| `selected=(string 'off')`| (initial, warning, ssr warning)| `<boolean: true>` |
+| `selected=(string 'on')`| (initial, warning)| `<boolean: true>` |
+| `selected=(string 'off')`| (initial, warning)| `<boolean: true>` |
```

Comment:
React 17 only had a warning on the client: https://codesandbox.io/s/react-17-option-selected-on-5nl1q
React 18 has a warning on client and server: https://codesandbox.io/s/react-18-option-selected-on-9ciud

I think the "ssr warning" is a bit misleading here in my opinion. If the snapshot reads "ssr warning I would expect a warning on the server. But the opposite is the case here.

Assuming that the other "ssr warning" changes follow the same pattern so not investigating those further